tests/antithesis/test-template/robustness/main.go

```diff
@@ -82,7 +82,7 @@ func testRobustness(ctx context.Context, hosts []string, baseTime time.Time, dur
 	if err != nil {
 		lg.Error("Failed to save visualization", zap.Error(result.Error))
 	}
-	assert.Always(result.Error == nil, "Robustness validation failed", map[string]any{"error": result.Error})
+	assert.Always(result.Error == nil, "Robustness validation passes", map[string]any{"error": result.Error})
 	lg.Info("Completed robustness validation")
```

Comment:
```suggestion
	lg.Info("Completed robustness validation", zap.Error(result.Error))
```

Reply:
Assertion is just for antithesis system, by itself doesn't show up in logs. This means if we run it locally we will not see the error.

Reply:
Can you fix this?